courses/views_test.py

```diff
@@ -90,6 +106,79 @@ def test_create_course_accepts_instructors(self):
         self.assertEqual(resp.status_code, 201, msg=resp.content)
         assert EdxAuthor.objects.count() == 2
 
+    def test_get_course_list(self):
+        """
+        The course list API should list all courses available.
+        """
+        course = CourseFactory.create()
+        resp = self.client.get(reverse('course-list'))
+        assert resp.status_code == 200, resp.content.decode('utf-8')
+        course_list = json.loads(resp.content.decode('utf-8'))
```

Comment:
`resp.json` should work, given this is django 1.9 https://docs.djangoproject.com/en/1.9/topics/testing/tools/#django.test.Response.json